release_notes.md

```diff
@@ -9,6 +9,13 @@ This file tracks release notes for the loop client.
 * All of the items under the "Next Release" heading should be included in the release notes.
 * As part of the PR that bumps the client version, the "Next Release" heading should be replaced with the release version including the changes.
 
+## Next release
+
+#### NewFeatures
```

Comment:
nit/ask: do you mind copying the `Breaking Changes` and `Bug Fixes` headings in as well? Can see this getting inconsistent over time if we don't copy them across with each new release.

Reply:
Sure thing!